baker/algolia/indexToAlgolia.ts

```diff
@@ -88,10 +83,24 @@ const indexToAlgolia = async () => {
         }
     }
 
-    index.replaceAllObjects(records)
-
     await wpdb.singleton.end()
     await db.closeTypeOrmAndKnexConnections()
```

Comment:
nit: seems like these two terminating functions belong to the main script (`indexToAlgolia`) rather than the `getPagesRecords` helper function? 

It seems like this helper function should be callable multiple times, and it isn't if it closes all connections.

Reply:
yes that makes sense, thanks!